projects/plugins/jetpack/tests/php/core-api/wpcom-endpoints/test-class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -452,15 +454,21 @@ public function menu_item_update_data() {
 			array(
 				'Finally some updates <span class="update-plugins count-5"><span class="update-count">5</span></span>',
 				array(
-					'count' => 5,
 					'title' => 'Finally some updates',
```

Comment:
Does it matter that these are in a different order?

Reply:
Not sure, running `yarn docker:phpunit --filter WP_Test_WPCOM_REST_API_V2_Endpoint_Admin_Menu` is successful so I think we are OK!

Reply:
Yeah, I had to swap the order of `count` and `title` in the `array()` because it was failing due to the expected and actual result having different ordered items.

I was just wondering if these orders were significant at all? UI regression tests seem fine as well.